tests/unit-tests/gconstruct/test_construct_graph.py

```diff
@@ -836,6 +861,37 @@ def check_regression_no_split(res):
     assert np.sum(res['val_mask']) == 1
     assert np.sum(res['test_mask']) == 1
 
+    # Check link prediction on customed masks
```

Comment:
ditto

Reply:
Also can you add a test case for link prediction when multiple edge types has their customer splits.

Reply:
It looks a little weird to write the tests for multiple edge types here. I have drafted another e2e tests for the link prediction with customized label split.

Reply:
And I have moved the link prediction test out to another function.